config/config.exs

```diff
@@ -6,8 +6,9 @@
 use Mix.Config
 
 # General application configuration
-config :ucx_ucc,
-  ecto_repos: [UcxUcc.Repo]
+config :ucx_ucc, UcxUcc.Repo,
```

Comment:
Had to make this change for a compile error through rpm script during mix release.

log_trace:
Check your digested files at "priv/static"
+ MIX_ENV=prod
+ mix release
warning: the VM is running with native name encoding of latin1 which may cause Elixir to malfunction as it expects utf8. Please ensure your locale is set to UTF-8 (which can be verified by running "locale" in your shell)
Compiling 363 files (.ex)

== Compilation error in file lib/ucx_ucc/repo.ex ==
** (ArgumentError) missing :adapter configuration in config :ucx_ucc, UcxUcc.Repo
    lib/ecto/repo/supervisor.ex:50: Ecto.Repo.Supervisor.compile_config/2
    lib/ucx_ucc/repo.ex:2: (module)
    (stdlib) erl_eval.erl:670: :erl_eval.do_apply/6